source/install/desktop.rst

```diff
@@ -85,3 +85,6 @@ Desktop App window is black and doesn't load the page
     - Finally, try disabling GPU hardware acceleration by using the `--disable-gpu <http://peter.sh/experiments/chromium-command-line-switches/#disable-gpu>`_ Chromium command line switch.
 
     If none of the above steps resolve the issue, please open a new ticket in the `Mattermost Troubleshooting Forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`_. 
+
+Desktop app appears off-screen after disconnecting from a secondary screen
+    Quit the app and delete the following file: ``USERPROFILE%\AppData\Roaming\Mattermost\bounds-info.json``
```

Comment:
Is this intended for end users? If so, then they need more explicit instructions on how to delete the file.

How does the end user distinguish between the app being off-screen and having crashed?

What do you mean by off-screen? Is it a multiple monitor setup? Or like Linux where you can have multiple workspaces, each with its own set of open windows?